test/metabase/api/card_test.clj

```diff
@@ -1838,6 +1839,167 @@
                   (name->position (:data (mt/user-http-request :crowberto :get 200 (format "collection/%s/items" coll-id-2)
                                                                :model "card" :archived "false"))))))))
 
+;;; +----------------------------------------------------------------------------------------------------------------+
+;;; |                                           PARAMETER VALUES ENDPOINTS                                           |
+;;; +----------------------------------------------------------------------------------------------------------------+
+
+(defn- do-with-chain-filter-fixtures
```

Comment:
same comment about `chain-filter` terminology as above. it's a bit confusing if we're not actually supporting chain filters in the code

I think `param-values` would work for all these cases